package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "ainft-js",
+  "version": "0.0.0",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "engines": {
+    "node": ">=16"
+  },
+  "scripts": {
+    "build": "rm -rf ./dist && tsc",
+    "test": "jest"
+  },
+  "files": [
+    "/dist"
+  ],
+  "repository": "git@github.com:ainft-team/ainft-js.git",
+  "keywords": [
+    "ainetwork",
+    "ain",
+    "TypeScript",
+    "AINFT",
+    "NFT",
+    "SDK"
+  ],
+  "author": "Crypto X Team",
```

Comment:
minor) as it's a public repo, what about something like 'AINFT Team'?

Reply:
Right, it makes sense to use the same name as the org. Thanks!